custom_components/tuya_local/devices/outdoor_inc_zipblind.yaml

```diff
@@ -0,0 +1,56 @@
+name: Outdoor Zip Blind
+products:
+  - id: f91miyhj
+    manufacturer: Outdoor Inc
+    model: DC1545BE
+entities:
+  - entity: cover
+    class: blind
+    current_position_dp: 8
+    set_position_dp: 9
+    dps:
+      - id: 1
+        name: control
+        type: string
+        mapping:
+          - dps_val: open
+            value: open
+          - dps_val: close
+            value: close
+          - dps_val: stop
+            value: stop
+      - id: 9
+        name: position
+        type: integer
+        range:
+          min: 0
+          max: 100
+      - id: 8
+        name: current_position
+        type: integer
+        range:
+          min: 0
+          max: 100
+      - id: 3
+        name: state
```

Comment:
Avoid the name "state" for attributes, as it clashes with the HA concept of entity state. Usually tuya uses "work_state", so you can use that.